python/ray/serve/tests/unit/test_pow_2_replica_scheduler.py

```diff
@@ -1853,7 +1853,7 @@ async def test_locality_aware_backoff_skips_sleeps(pow_2_scheduler):
     s.update_replicas([r1, r2, r3])
 
     # The request should be served by r3 without added latency.
-    done, _ = await asyncio.wait([task], timeout=0.01)
+    done, _ = await asyncio.wait([task], timeout=1)
```

Comment:
Leave a comment that this would fail due to the extremely high backoff sequence that's set